terra-batch-libchart/templates/_wds.tpl

```diff
@@ -26,7 +26,7 @@ spec:
               mountPath: {{ .Values.wds.conf_dir }}/{{ .Values.wds.conf_file }}
               subPath: {{ .Values.wds.conf_file }}
           env:
-            - name: SPRING_CONFIG_LOCATION
+            - name: SPRING_CONFIG_ADDITIONAL-LOCATION
```

Comment:
I think `SPRING_CONFIG_LOCATION` replaces the default config location in the classpath, whereas `SPRING_CONFIG_ADDITIONAL-LOCATION` adds an additional location. See: https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config.files
 
It feels like the latter is more what we want, so we can keep `classpath:application.yaml` as a default and only put environment-specific configs in the helm chart.

Q: should C-BAS do similar here?

Reply:
interesting - makes sense to me